docker-compose/docker-compose.yml

```diff
@@ -25,19 +25,19 @@ services:
       - 18080:18080/tcp
     volumes:
       - monero:/home/monero/.bitmonero:rw
-      - /dev/null:/home/monero/.bitmonero/bitmonero.log:rw
       - /dev/hugepages:/dev/hugepages:rw
     restart: unless-stopped
     command: >-
       --zmq-pub tcp://0.0.0.0:18083
       --disable-dns-checkpoints
-      --enable-dns-blocklist
```

Comment:
Any specific reason for removing it?

Reply:
I would leave it. I had to add an additional list of blocked IP addresses as my monerod fell behind 12 blocks but it thought it was still 100% on main net.

Reply:
Monero documentation says it is "Legacy - These options should no longer be necessary. They are still present in monerod for backwards compatibility."   

https://monerodocs.org/interacting/monerod-reference/#legacy

I have no problem leaving it if thats the consensus.  Please advise.

Reply:
I'm not sure why it's "legacy" and who categorized it. `--zmq-pub` is also under legacy section there and it's definitely not.

Reply:
Ok.  I'll re-add it and push soon.  I always struggle with git command to amend or squish or whatever.  Wish me luck....or suggest the command if you have time.

Reply:
I googled it :)
amend is easy, its the squish that confuses me.